cernopendata_client/cli.py

```diff
@@ -81,7 +83,7 @@ def get_record(server, recid, doi, title, output_fields):
 @click.option("--title", help="Record title")
 @click.option(
     "--protocol",
-    default="root",
+    default="http",
     type=click.Choice(["root", "http"]),
```

Comment:
Please order alphabetically, `type=click.Choice(["http", "root"]),` here and in the other function.